WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,12 +1,19 @@
-What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.
+* [*] Signup: Fixed bug where username selection screen could be pushed twice. [#17624]
```

Comment:
Just double-checking, given that everything login-related in Jetpack might not always be applicable…

I'm often confusing Signup and Signin but IINM Signup is about account creation (not logging in using existing account), which is something that's not yet applicable to Jetpack? Except if this is related to the recent addition to SIWA support in Jetpack (which can lead to account creation in that case)?

cc @leandroalonso

Reply:
> Except if this is related to the recent addition to SIWA support in Jetpack (which can lead to account creation in that case)?

Yes, you're right. Although I'd consider any account creation not part of Jetpack, given if a user creates an account there they can't really do much (site creation is not enabled).

Reply:
Right, that's what I thought.

Given that's "not really part of Jetpack" (even if _technically_ they can create an account via SIWA, it's mostly a side effect of SIWA support but they can't really do much anyway), should we really include this into the Release Notes then? Or would it make more sense to remove it?

Reply:
Let's remove it. :)